paymentsheet/src/test/java/com/stripe/android/paymentsheet/state/DefaultPaymentSheetLoaderTest.kt

```diff
@@ -1076,28 +1084,27 @@ internal class DefaultPaymentSheetLoaderTest {
             ).getOrThrow()
 
             assertThat(attemptedToRetrievePaymentMethods).isFalse()
-            assertThat(state.customerPaymentMethods).containsExactlyElementsIn(cards)
+
+            assertThat(state.customer).isEqualTo(
+                CustomerState(
+                    id = "cus_1",
+                    ephemeralKeySecret = "ek_123",
+                    paymentMethods = cards,
+                )
+            )
         }
 
     @OptIn(ExperimentalCustomerSessionApi::class)
     @Test
-    fun `When 'CustomerSession' config is provided but no customer object was returned, should not fetch and return no payment methods`() =
+    fun `When 'CustomerSession' config is provided but no customer object was returned, should report error and return error`() =
```

Comment:
Do we have this test for both livemode + testmode? The behavior should be different between them.